core/src/main/java/com/linecorp/armeria/server/encoding/HttpEncoders.java

```diff
@@ -76,20 +88,27 @@ private static HttpEncodingType determineEncoding(String acceptEncoding) {
             }
             if (encoding.contains("*")) {
                 starQ = q;
+            } else if (encoding.contains("br") && q > brQ) {
+                brQ = q;
             } else if (encoding.contains("gzip") && q > gzipQ) {
                 gzipQ = q;
             } else if (encoding.contains("deflate") && q > deflateQ) {
                 deflateQ = q;
             }
         }
-        if (gzipQ > 0.0f || deflateQ > 0.0f) {
-            if (gzipQ >= deflateQ) {
+        if (brQ > 0.0f || gzipQ > 0.0f || deflateQ > 0.0f) {
+            if (brQ != -1.0f && brQ >= gzipQ) {
```

Comment:
IIUC, Brotli is better at static data because of its compression ratio and
GZIP is better at dynamic data. If Brotli and GZIP have the same priority,
since Armeria is mostly used for REST API and RPC, wouldn't it make sense to prioritize GZIP?

Reply:
That makes sense. But I'd like to hear other people's opinions. @minwoox @trustin

Reply:
Had a chat with @minwoox and @trustin. 
We decided to respect the order of encoding if some encodings have the same weight.
For example:
```
// deflate will be chosen
Accept-Encoding: deflate, gzip, *;q=0.5
// gzip will be chosen
Accept-Encoding: deflate;q=0.5, gzip;0.9, br;q=0.9
```

Reply:
Ok. I'll implement it. Thank you for comment. 😀